modules/ROOT/pages/authentication-authorization/index.adoc

```diff
@@ -45,11 +45,23 @@ A plugin option for building custom integrations.
 It is recommended that this option is used as part of a custom delivery as negotiated with link:https://neo4j.com/professional-services/[Neo4j Professional Services].
 For more information, see link:{neo4j-docs-base-uri}/java-reference/{page-version}/extending-neo4j/security-plugins#extending-neo4j-security-plugins[Java Reference -> Authentication and authorization plugins].
 
-
 *Kerberos authentication and single sign-on*::
 In addition to LDAP, native, and custom providers, Neo4j supports Kerberos for authentication and single sign-on.
 Kerberos support is provided via the link:{neo4j-docs-base-uri}/kerberos-add-on/current/[Neo4j Kerberos Add-On].
 
+
+In addition, Neo4j supports mixed-mode authentication and authorization that allows to use multiple authentication providers in your database setup.
```

Comment:
I am not sure if we need all this information here on the index page where we just list the providers. I think just something like this would suffice:
```suggestion
Neo4j also supports mixed-mode authentication that allows you to use multiple authentication providers in your database setup.
For more information and examples, see xref:authentication-authorization/ldap-integration.adoc#auth-ldap-configure-provider[Set Neo4j to use LDAP] and xref:authentication-authorization/sso-integration.adoc#auth-sso-configure-sso[Configure Neo4j to use OpenID Connect].
```